src/content/reference/okteto-manifest.mdx

```diff
@@ -111,22 +113,45 @@ dependencies:
     timeout: 15m
 ```
 
-Or:
+#### repository (string)
```

Comment:
I would use the same format that we use in the [build section](https://www.okteto.com/docs/reference/okteto-manifest/#build-object-optional). We put the sample first, and then a list of items explaining each field